project/appweb/src/main/java/web/vasilizas/UrlRepository.java

```diff
@@ -0,0 +1,14 @@
+package web.vasilizas;
+
+import java.util.Map;
+
+public class UrlRepository {
+    public static final Map<String, String> urlMap = Map.of(
+            "Student", "/myweb/student/student.html",
+            "Admin", "/myweb/admin/admin.html",
+            "Teacher", "/myweb/teacher/teacher.html",
+            "Error", "/myweb/error/auth-error.html");
+
+    private UrlRepository() {
+    }
```

Comment:
Статические анализаторы, такие, как SonarQube обычно помечают методы и конструкторы с пустым телом, как ненужные, поэтому лучше туда добавлять комментарий, описывающий, почему метод или конструктор пустой
```suggestion
    private UrlRepository() {
        // blank private constructor for utility class
    }
```

Reply:
исправил. но вроде бы сонар ругается если конструктор публичный, на приватный он не ругается